src/payment/payment-action-creator.ts

```diff
@@ -53,15 +53,21 @@ export default class PaymentActionCreator {
         instrumentId?: string,
         shouldSaveInstrument?: boolean,
         target?: string,
-        promise?: Promise<undefined>
+        promise?: Promise<undefined>,
+        setAsDefaultInstrument?: boolean
```

Comment:
Just a cosmetic .. can we move this new parameter to be right after `shouldSaveInstrument` as they are closely related? I think this would be more organised, better for method usability and easier for future updates.

Reply:
Done, though arguably I should've done it in a different PR, muddies this one a bit. I can split it out if this one is now to hard to read.

Reply:
It's not hard to read for me now with the new order, thanks.